packages/react/jest.setup.ts

```diff
@@ -1,4 +1,26 @@
-import * as enzyme from 'enzyme';
-import * as Adapter from 'enzyme-adapter-react-16';
+import '@testing-library/jest-dom/extend-expect';
 
-enzyme.configure({ adapter: new Adapter() });
+// jsdom doesn't implment IntersectionObserver
+// https://github.com/jsdom/jsdom/issues/2032
+if (typeof window !== 'undefined') {
+  const mockObserverAPI = () =>
+    jest.fn().mockReturnValue({
+      observe: jest.fn(),
+      unobserve: jest.fn(),
+      disconnect: jest.fn(),
+    });
+
+  window.ResizeObserver = mockObserverAPI();
+  window.IntersectionObserver = mockObserverAPI();
+
+  // jest implements getBoundingClientRect, but not DOMRect
+  if (!window.DOMRect) {
```

Comment:
@mathisscott I was able to fix the popover/dropdown issues by mocking DOMRect (used in popover.element.ts). Another thing jsdom doesn't implement...

Unfortunately, I couldn't get past typescript, hence the `@ts-ignore` below...

Reply:
That's awesome! 

It's so weird what jsdom does/does not include...